src/main/java/graphql/nadel/TypeDefinitionsWithRuntimeWiring.java

```diff
@@ -0,0 +1,56 @@
+package graphql.nadel;
+
+import graphql.schema.idl.TypeDefinitionRegistry;
+import graphql.schema.idl.TypeRuntimeWiring;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * Stores {@link TypeDefinitionRegistry} and collection of {@link TypeRuntimeWiring}s.
+ */
+public class TypeDefinitionsWithRuntimeWiring {
```

Comment:
We should all classes annotate with PublicAPI or Internal (from graphql-java).

Reply:
will do